nearlib/API.md

```diff
@@ -108,7 +108,7 @@ Generate a new keypair from a random seed
 #### Examples
 
 ```javascript
-const keyWithRandomSeed = await KeyPair.fromRandomSeed();
```

Comment:
nit: it's generally better to keep APIs with potentially heavy compute as async (cause then you can replace implementation on something that doesn't block threads)

Reply:
Oh, I see. The reason for this change was keep API the same as WalletAccount. But I can just wrap logic with `.then(...)`